### PR TITLE
Add CHANGELOG.md

### PR DESCRIPTION
**Pull request by laura-ding**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## v1.0.0rc4(2020-03-23)
+Compatible the 1.0.0-rc4 version of nebula-graph
```
Comment (jude-zhu) [inline CHANGELOG.md:2]: Compatible with (*￣︶￣)

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+- New features
+	- None
+- Change
+	- Delete the parameter `is_async` in `ConnectionPool`
+	- Move `AuthException`, `ExecutionException` and `SimpleResponse` from `nebula/Client.py` to `nebula/Common.py`.  You need to add `from nebula.Common import *` in your project.
+
+## v1.0.0-rc2-1(2019-12-05)
+Compatible the 1.0.0-rc2 and 1.0.0-rc3 version of nebula-graph
```
Comment (jude-zhu) [inline CHANGELOG.md:11]: ditto

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+- New features
+	- None
+- Change
+	- Update thrift
+
+## v1.0.0rc1(2019-11-27)
+Compatible the 1.0.0-rc1 version of nebula-graph
```
Comment (jude-zhu) [inline CHANGELOG.md:19]: ditto

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+## v1.0.0rc4(2020-03-23)
+Compatible with the 1.0.0-rc4 version of nebula-graph
+
+- New features
+	- None
+- Change
```
Comment (Amber1990Zhang) [inline CHANGELOG.md:6]: Changes

**Pull request merged**